lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -536,6 +570,7 @@ def main():
     cfn.describe_stacks = backoff_wrapper(cfn.describe_stacks)
     cfn.list_stack_resources = backoff_wrapper(cfn.list_stack_resources)
     cfn.delete_stack = backoff_wrapper(cfn.delete_stack)
+    cfn.update_termination_protection = backoff_wrapper(cfn.update_termination_protection)
```

Comment:
We can't just assume people have the latest boto3, so when using newer features we have to add a check for the function being available, such as `if hasattr(cfn, 'update_termination_protection')` and then fail if that feature is missing and the user specified `termination_protection`. If they didn't set `termination_protection` and are on an older `boto3` they still should be able to use the module.

Reply:
agreed. I will add that in. Thanks for the review.